packages/visual-editor/src/hooks/useComponentProps.tsx

```diff
@@ -189,5 +189,9 @@ export const useComponentProps = ({
     ...omit(props, CF_STYLE_ATTRIBUTES, ['cfHyperlink', 'cfOpenInNewTab']),
   };
 
+  if (definition.children) {
```

Comment:
Hey @chasepoirier did you check whether this affects assembly noides? We set `children: true` for them to render their children within an experience but they should not be able to accept new ones. So it's rather a fixed set of children. I suspect that we need to check here that the definition is not an assembly one.

Reply:
I believe Assembly nodes still function as expected after this PR. The reason being because I had modified the component used for DesignComponents which looks like this:

```ts
// Feel free to do any magic as regards variable definitions for design components
// Or if this isn't necessary by the time we figure that part out, we can bid this part farewell
export const DesignComponent: React.FC<DesignComponentProps> = (props) => {
  if (props.editorMode) {
    const { node } = props;

    return props.renderDropzone(node, {
      ['data-test-id']: 'contentful-container',
      ['data-cf-node-id']: node.data.id,
      ['data-cf-node-block-id']: node.data.blockId,
      ['data-cf-node-block-type']: node.type,
      id: 'design-component',
      className: props.className,
      style: designComponentStyle,
    });
  }
  // Using a display contents so design component content/children
  // can appear as if they are direct children of the div wrapper's parent
  return <div data-test-id="design-component" {...props} style={designComponentStyle} />;
};
```

Since the return is a `renderDropzone` instead of the `children` prop in editorMode the changes made don't have an effect